browser-interface/packages/kernel-web-interface/dcl-crypto.ts

```diff
@@ -0,0 +1,52 @@
+/**
+ * @public
+ */
+export type Signature = string
+
+/**
+ * @public
+ */
+export type EthAddress = string
+
+/**
+ * @public
+ */
+export type IdentityType = {
+  privateKey: string
+  publicKey: string
+  address: string
+}
+
+/**
+ * @public
+ */
+export type AuthChain = AuthLink[]
+
+/**
+ * @public
+ */
+export type AuthLink = {
+  type: AuthLinkType
+  payload: string
+  signature: Signature
+}
+
+/**
+ * @public
+ */
+export enum AuthLinkType {
```

Comment:
All the items in this file can be imported from `@dcl/schemas`

Reply:
Agreed, but I'd rather keep this as "move-only" -- this Pull request is already big enough, I'll create another issue